.github/actions/deploy-happy-stack/action.yml

```diff
@@ -46,7 +51,12 @@ runs:
         IMAGE_TAG="${TAG:-sha-$GITHUB_SHA}"
         echo "Targetting Stack ${STACK_NAME}, tagging with '${IMAGE_TAG}'"
 
-        if $(happy list --aws-profile "" --env=${ENV} 2>&1 | grep -q ${STACK_NAME} ); then
+        happy list --aws-profile "" --env=${ENV}
+
+        if ${DELETE_STACK}; then
+          echo "Deleting stack ${STACK_NAME}"
+          happy delete --aws-profile "" --env=${ENV} ${STACK_NAME}
+        elif $(happy list --aws-profile "" --env=${ENV} 2>&1 | grep -q " ${STACK_NAME} " ); then
```

Comment:
add the grep expression to the description as well, including the reason why it was changed

Reply:
done!